airsonic-main/src/main/java/org/airsonic/player/service/SecurityService.java

```diff
@@ -166,13 +166,24 @@ public User getUserByEmail(String email) {
      * Returns whether the given user has administrative rights.
      */
     public boolean isAdmin(String username) {
-        if (User.USERNAME_ADMIN.equals(username)) {
-            return true;
-        }
         User user = getUserByName(username);
         return user != null && user.isAdminRole();
     }
 
+    /**
+     * Returns username with admin rights.
+     *
+     * @return username of admin user
+     */
+    public String getAdminUsername() {
+        for (User user : userDao.getAllUsers()) {
```

Comment:
Minor thing: we should probably make sure the order is consistent (order by id or user name), I'm not sure of the behavior of all databases, and this might be confusing if the order somehow changes between runs.